web/core/components/issues/issue-layouts/filters/header/filters/custom-properties.tsx

```
@@ -150,28 +182,32 @@ export const FilterCustomProperty: React.FC<Props> = observer((props) => {
-                        {t("view_more")}
-                      </button>
-                    ) : null}
-                    {
-                      properties.length == 0 ?
-                      <p className="text-xs italic text-custom-text-400">{t("no_matches_found")}</p> : null
-                    }
+                      {(groupedSection.page ?? 1) < (groupedSection.total_pages ?? 1) && properties.length ? (
+                        <button
+                          onClick={() => fetchNextPage(groupKey)}
+                          className="ml-8 text-xs font-medium text-custom-primary-100 cursor-pointer"
+                        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)